traitsui/tests/test_editor.py

```diff
@@ -937,10 +937,6 @@ def test_sync_value_both(self):
     # regression test for enthought/traitsui#1543
     @requires_toolkit([ToolkitName.qt])
     @unittest.skipIf(no_modal_dialog_tester, "ModalDialogTester unavailable")
-    @unittest.skipIf(
-        is_mac_os,
-        "There is a separate issue on OSX. See enthought/traitsui#1550"
-    )
     def test_editor_error_msg(self):
```

Comment:
This test is _very_ similar to the newly added range editor test `test_modify_out_of_range`.  However, this test was motivated by the text format of the error dialog.  The other is about the range editor not triggering two error dialogs.  I think it make sense to leave them both...